docs/sidebars.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     },
     {
       type: "category",
-      label: "Getting Started",
+      label: "Get Started",
```

Comment:
Why should the label (get started) be different from the url slug and title (getting started)?

Reply:
This is being changed in main into another PR